src/test/librbd/fsx.cc

```diff
@@ -323,7 +324,7 @@ int register_journal(rados_ioctx_t ioctx, const char *image_name) {
                 return r;
         }
 
-        journal::Journaler journaler(io_ctx, image_id, JOURNAL_CLIENT_ID, 0);
+        journal::Journaler journaler(io_ctx, image_id, JOURNAL_CLIENT_ID, {});
```

Comment:
I ratcheted it down in the cases where the test case truly depended on looking at the commit position.  But in general, this parameter only has any effect after a crash.